packages/semaphore-signature-pcd/src/SemaphoreSignaturePCD.ts

```diff
@@ -63,12 +64,24 @@ export interface SemaphoreSignaturePCDClaim {
    * Stringified `BigInt`.
    */
   identityCommitment: string;
-}
 
-export interface SemaphoreSignaturePCDProof {
-  proof: FullProof;
+  /**
+   * Stringified `BigInt`.
+   */
+  nullifierHash: string;
+
+  /**
+   * Stringified `BigInt`.
+   */
+  externalNullifier: string;
+
+  /**
+   * Stringified `BigInt`.
+   */
+  signal: string;
```

Comment:
We should be able to leave this out of the claim if we have `signedMessage`; if the intended behavior is always that the signal is the output of `generateMessageHash` on `signedMessage` then we should just compute that when we generate/verify proofs.